solr/solrj/src/java/org/apache/solr/client/solrj/impl/BaseHttpClusterStateProvider.java

```diff
@@ -22,12 +22,7 @@
 import java.io.IOException;
 import java.lang.invoke.MethodHandles;
 import java.time.Instant;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
I think we normally list every import!!!

Reply:
IntellJ did this - I ran ./gradle tidy and noticed it complained too, fixed in the recent commit

Reply:
I recommend using the following IntelliJ code style: https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml
"Tidy" only manipulates whitespace; it does not adjust imports, unlike that IntelliJ code style config.
Ideally this suggestion would be listed in `dev-docs/IDEs.adoc`